test/common/schemaform/review/PhoneNumberWidget.unit.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { expect } from 'chai';
+import SkinDeep from 'skin-deep';
+
+import PhoneNumberWidget from '../../../../src/js/common/schemaform/review/PhoneNumberWidget';
+
+describe('Schemaform review <PhoneNumberWidget>', () => {
+  it('should format phone number', () => {
+    const tree = SkinDeep.shallowRender(
+      <PhoneNumberWidget value="1234567890"/>
+    );
+
+    expect(tree.text()).to.equal('(123) 456-7890');
+  });
+  it('should render empty value', () => {
+    const tree = SkinDeep.shallowRender(
+      <PhoneNumberWidget/>
+    );
+
+    expect(tree.text()).to.equal('');
```

Comment:
This is actually a bug. It should equal `undefined`, because JSON Schema interprets empty strings as non-empty fields. There's an issue to fix this with the SSN widget, too.